react-frontend/public/config.json

```diff
@@ -0,0 +1,3 @@
+{
+  "strapiApiUrl": "http://localhost:1337/graphql"
```

Comment:
Would strapi and react-frontend be on the same machine?

Reply:
They are not when the app runs in openshift, but they are when we are doing local development (2 different docker containers, but the ports exposed are 3000 and 1337).  The dc.yaml file overwrites this default in openshift.  In local development that doesn't happen so react and strapi can still talk.